src/components/graph/NodeTooltip.vue

```diff
@@ -86,6 +86,11 @@ const onIdle = () => {
     ctor.title_mode !== LiteGraph.NO_TITLE &&
     canvas.graph_mouse[1] < node.pos[1] // If we are over a node, but not within the node then we are on its title
   ) {
+    if (comfyApp?.tooltipCallback?.(node, 'DESCRIPTION')) {
```

Comment:
You can setup a pinia store to store/track the currently hovered node and node part instad of doing it via a callback. Setting up a pinia store will allow this state to be observable by multiple components if we need it later.

Reply:
Done. The means of hiding tooltips for a node actively displayed in the documentation sidebar tab was concerning, but works better than expected.